packages/cosmic-swingset/x/swingset/internal/types/msgs.go

```diff
@@ -155,9 +150,6 @@ func (msg MsgProvision) ValidateBasic() error {
 	if len(msg.Nickname) == 0 {
 		return sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, "Nickname cannot be empty")
 	}
-	if msg.Address.Empty() {
```

Comment:
This is just an unnecessary repeated verification (already performed above in the current func).